server/src/main/java/org/candlepin/dto/api/v1/DistributorVersionTranslator.java

```diff
@@ -60,28 +63,38 @@ public DistributorVersionDTO populate(DistributorVersion source, DistributorVers
     public DistributorVersionDTO populate(ModelTranslator modelTranslator, DistributorVersion source,
         DistributorVersionDTO dest) {
 
-        dest = super.populate(modelTranslator, source, dest);
+        if (source == null) {
+            throw new IllegalArgumentException("source is null");
+        }
 
-        dest.setId(source.getId());
-        dest.setName(source.getName());
-        dest.setDisplayName(source.getDisplayName());
+        if (dest == null) {
+            throw new IllegalArgumentException("dest is null");
+        }
 
-        if (modelTranslator != null) {
+        dest.id(source.getId())
+            .name(source.getName())
+            .displayName(source.getDisplayName())
+            .created(Util.toDateTime(source.getUpdated()))
```

Comment:
This is setting the source's updated date to the dest's created date. it should be setting the created date

Reply:
Done.